src/components/TodoForm.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+import { useTodoContext } from '../context/TodoContext';
+import { addTodo } from '../api/todos';
+import { Error } from '../types/Error';
+
+export const TodoForm: React.FC = () => {
+  const {
+    setTodos,
+    setError,
+  } = useTodoContext();
+
+  const [todoTitle, setTodoTitle] = useState('');
+  const [isCreating, setIsCreating] = useState(false);
+
+  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (!todoTitle) {
+      setError(Error.INVALID_INPUT);
+
+      return;
+    }
+
+    setIsCreating(true);
+
+    const newTodo = {
+      id: 0,
+      userId: 10407,
+      title: todoTitle,
```

Comment:
U can create another file for important variables and use it with export. 
Oh sorry, I see this variable in App.tsx